Makefile.am

```diff
@@ -108,7 +108,25 @@ libuv_la_SOURCES += src/unix/async.c \
 endif  # WINNT
 
 EXTRA_DIST = test/fixtures/empty_file \
-             test/fixtures/load_error.node
+             test/fixtures/load_error.node \
+             src/unix/uv-dtrace.d \
+             include \
```

Comment:
Is this needed? AFAIS stdint-msvc2008.h and uv-aix.h are missing from the tarball, I wonder why.